cbv_spiketest.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Mar 18 10:12:05 2019
+
+@author: mikkelnl
+"""
+
+from __future__ import division, with_statement, print_function, absolute_import
+from six.moves import range
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import signal
+from scipy.signal import savgol_filter, medfilt
+from scipy.interpolate import interp1d
+import astropy.units as u
+from scipy.interpolate import pchip_interpolate
+
+from corrections.cbv_corrector.cbv_main import CBV
+import corrections
+
+cbv_area = 334
+#data_folder= '/home/mikkelnl/ownCloud/Documents/Asteroseis/TESS/TASOC_code/corrections/corrections/data/cbv_old'
+data_folder= '/home/mikkelnl/ownCloud/Documents/Asteroseis/TESS/TASOC_code/corrections/corrections/data/cbv'
```

Comment:
Is this code (`cbv_spiketest.py`) just a test on a local machine for @Miklnl ? If it is we should probably flag it as such in the docstring at the top of the script.

Reply:
Yes... But it is fixed in the newest version of the devel branch.... The issue is really that this PR should not change anything not having something to do with the ensemble code... So any changes to the CBV code, `BaseCorrector`, `TaskManager` or `run_tesscorr` should properly be reverted

Reply:
Got it!